BarrysPeanutsAkkaFlat/src/main/java/barryspeanuts/actor/CustomerActor.java

```diff
@@ -0,0 +1,83 @@
+package barryspeanuts.actor;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.AbstractBehavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.actor.typed.javadsl.Receive;
+import java.util.UUID;
+
+public class CustomerActor extends AbstractBehavior<Object> {
+
+  private CustomerActor(ActorContext<Object> context) {
+    super(context);
+  }
+
+  public static Behavior<Object> create() {
+    return Behaviors.setup(CustomerActor::new);
+  }
+
+  @Override
+  public Receive<Object> createReceive() {
+    return newReceiveBuilder()
+        .onMessage(CustomerActor.ShippingReceipt.class, this::handleShippingReceipt)
+        .onMessage(CustomerActor.PaymentReceipt.class, this::handlePaymentReceipt)
+        .build();
+  }
+
+  private Behavior<Object> handleShippingReceipt(CustomerActor.ShippingReceipt msg) {
+    getContext()
+        .getLog()
+        .info(
+            "Received shipping confirmation by id {} for purchaseId {}.\n",
```

Comment:
You are inconsistent with whether you punctuate the end of your logs (trailing punctuation not really needed, but not a big deal though, mentioned before at https://github.com/reselbob/WorkingWithTheActorModel/pull/1#discussion_r1291427243).

Reply:
Addressed. I think I caught them all.